sdcm/nemesis.py

```diff
@@ -1370,16 +1370,23 @@ def disrupt_modify_table(self):
 
     def disrupt_mgmt_backup(self):
         self._set_current_disruption('ManagementBackup')
-        # TODO: When cloud backup is supported - add this to the belowq 'if' statement:
+        # TODO: When cloud backup is supported - add this to the below 'if' statement:
         #  and not self.cluster.params.get('use_cloud_manager', default=None)
         if not self.cluster.params.get('use_mgmt', default=None):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
         if not self.cluster.params.get('backup_bucket_location'):
             raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
-        bucket_location_name = self.cluster.params.get('backup_bucket_location').split()
-        mgr_task = mgr_cluster.create_backup_task(location_list=['s3:{}'.format(bucket_location_name[0])])
+        cluster_backend = self.cluster.params.get('cluster_backend')
+        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
```

Comment:
in order not to confuse aws and gce bucket location names. perhaps we can consider splitting to 2 parameters with different prefixes like aws_backup_bucket_location and gce_backup_bucket_location

Reply:
I don't see the point of adding more parameters, it mean the same thing, and used the same.

Reply:
i agree with @fruch , as this is exactly same thing... we must avoid creating repeated variables, when they mean the same thing, and used the same way...